SabreTools.Hashing.Test/TestHelper.cs

```diff
@@ -216,6 +216,24 @@ internal static class TestHelper
 #endif
         };
 
+        public static readonly Dictionary<(string, string), int> _knownCompareHashes = new()
```

Comment:
Instead of this dictionary, you should look into using `InlineData` with tests. An example:
```c#
[Theory]
[InlineData("hash1", "hash2", 100)]
public void TestThing(string first, string second, int expected)
```